pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/DoubleAggregationResultHolder.java

```diff
@@ -61,6 +66,16 @@ public double getDoubleResult() {
     return _value;
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * @return
+   */
+  @Override
+  public int getIntResult() {
+    throw new RuntimeException("Method 'getLongResult' not supported for class " + getClass().getName());
```

Comment:
(minor) same for other places
```suggestion
    throw new RuntimeException("Method 'getIntResult' not supported for class " + getClass().getName());
```